content/security/for-admins/provisioning/group-mapping.md

```diff
@@ -0,0 +1,181 @@
+6. Assign users to the group(s) that you create.
+
+The next time you sync your groups with Docker, your users will map to the Docker groups you defined.
+
+{{< /tab >}}
+{{< tab name="Entra ID" >}}
+
+The user interface for your IdP may differ slightly from the following steps. You can refer to the [Entra ID documentation](https://learn.microsoft.com/en-us/azure/active-directory/app-provisioning/customize-application-attributes) to verify.
+
+To set up group mapping:
+
+1. Sign in to Entra ID and go to your application.
+2. Go to **Manage > Single sign-on**.
+3. Select **Add a group claim**.
+4. In **Group Claims**, select **Groups assigned to the application** with the source attribute **Cloud-only group display names (Preview)**.
+5. Select **Advanced options**, then the **Filter groups** option.
+6. Configure the attribute like the following:
+   - **Attribute to match**: `Display name`
+   - **Match with**: `Contains`
+   - **String**: `:`
+7. Select **Save**.
+8. Go to **Groups > All groups** then select **New group** to create your group(s).
+9. Assign users to the group(s) that you create.
+
+The next time you sync your groups with Docker, your users will map to the Docker groups you defined.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+### Use group mapping with SCIM
+
+The following steps describe how to set up and use group mapping with SCIM. Before you begin, make sure you [set up SCIM](./scim.md#enable-scim) first.
+
+{{< tabs >}}
+{{< tab name="Okta" >}}
+
+The user interface for your IdP may differ slightly from the following steps. You can refer to the [Okta documentation](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-enable-group-push.htm) to verify.
+
+To set up your groups:
+
+1. Sign in to the Okta Console to go to your application.
+2. Select **Applications > Provisioning > Integration**.
+3. Select **Edit** to enable groups on your connection, then select **Push groups**.
+4. Select **Save**. Saving this configuration will add the **Push Groups** tab to your application.
+5. Create your groups by navigating to **Directory > Groups**.
+6. Add your groups using the format `organization:team` that matches the names of your organization(s) and team(s) in Docker.
+7. Assign users to the group(s) that you create.
+8. Return to **Applications > Provisioning> Integration**, then select the **Push Groups** tab to open the view where you can control and manage how groups are provisioned.
+9. Select **Push Groups > Find groups by rule**.
+10. Configure the groups by rule like the following:
+    - Enter a rule name, for example `Sync groups with Docker Hub`
+    - Match group by name, for example starts with `docker:` or contains `:` or multi-organization
+    - If you enable **Immediately push groups by rule**, sync will happens as soon as there's a change on the group or assignments on the group. Enable this if you don't want to manually push groups.
```

Comment:
question: is "assignments on the group" correct? It reads a bit odd but it might just be because I'm not familiar with the domain 

```suggestion
    - If you enable **Immediately push groups by rule**, sync will happen as soon as there's a change on the group or assignments on the group. Enable this if you don't want to manually push groups.
```

Reply:
I think "to the group" will make more sense 😄 will edit and streamline a bit!